synapse/handlers/room.py

```diff
@@ -1165,7 +1166,7 @@ async def send(
                 EventTypes.PowerLevels, pl_content, False
             )
             current_state_group = power_context._state_group
-            await send(power_event, power_context, creator)
```

Comment:
`send` only seems to be called once now -- can we inline it to simplify the code?

Reply:
Good call. [0b36b9e](https://github.com/matrix-org/synapse/pull/14228/commits/0b36b9e9f0cb123aac54fc6b863827d2466382a1)